test/e2e/resources/full-chart-install.yaml

```diff
@@ -0,0 +1,6775 @@
+---
+# Source: cluster-api-operator/templates/operator-components.yaml
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can this file be updated as a part of `make generate`? I assume the tests will break each time we add a new API change. It can be done in a follow-up PR

Reply:
I fixed this in https://github.com/kubernetes-sigs/cluster-api-operator/pull/212